sample/src/main/java/com/timehop/stickyheadersrecyclerview/sample/RecyclerItemClickListener.java

```diff
@@ -27,7 +27,7 @@ public RecyclerItemClickListener(Context context, OnItemClickListener listener)
   @Override public boolean onInterceptTouchEvent(RecyclerView view, MotionEvent e) {
     View childView = view.findChildViewUnder(e.getX(), e.getY());
     if (childView != null && mListener != null && mGestureDetector.onTouchEvent(e)) {
-      mListener.onItemClick(childView, view.getChildPosition(childView));
+      mListener.onItemClick(childView, view.getChildAdapterPosition(childView));
```

Comment:
The sample app doesn't do anything that would cause a non-adapter view to show up as a child of the recycler view, so I didn't change it there on purpose.